src/Compilers/CSharp/Portable/Compilation/SyntaxTreeSemanticModel.cs

```diff
@@ -1644,6 +1644,18 @@ public override ISymbol GetDeclaredSymbol(VariableDeclaratorSyntax declarationSy
         }
 
         /// <summary>
+        /// Given an argument syntax, get symbol for an out var that it declares, if any.
+        /// </summary>
+        /// <param name="declarationSyntax">The syntax node that declares a variable.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The symbol that was declared.</returns>
+        public override ISymbol GetDeclaredSymbol(ArgumentSyntax declarationSyntax, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            var memberModel = this.GetMemberModel(declarationSyntax);
+            return memberModel == null ? null : memberModel.GetDeclaredSymbol(declarationSyntax, cancellationToken);
```

Comment:
Simplify to `return  memberModel?.GetDeclaredSymbol(declarationSyntax, cancellationToken);`?